libsabledb/src/commands/generic_commands.rs

```diff
@@ -170,6 +174,34 @@ impl GenericCommands {
         };
         Ok(Some(md.value_type()))
     }
+
+    async fn expire(
+        client_state: Rc<ClientState>,
+        command: Rc<RedisCommand>,
+        response_buffer: &mut BytesMut,
+    ) -> Result<(), SableError> {
+        let builder = RespBuilderV2::default();
+        // at least 2 arguments
+        check_args_count!(command, 2, response_buffer);
```

Comment:
Expire commands has the following syntax:

```
EXPIRE key seconds [NX | XX | GT | LT]
```
So we expect at least **3** arguments `expire`, `key`, `seconds`